crates/xline/src/conflict/mod.rs

```diff
@@ -1,15 +1,23 @@
+use std::sync::Arc;
+
 use curp::{
     cmd::Command as CurpCommand,
-    server::{SpObject, UcpObject},
+    server::{conflict::CommandEntry, SpObject, UcpObject},
 };
 use utils::interval_map::Interval;
-use xlineapi::{command::Command, interval::BytesAffine, RequestBackend, RequestWrapper};
+use xlineapi::{
+    command::{Command, KeyRange},
+    interval::BytesAffine,
+    RequestWrapper,
+};
+
+use crate::storage::lease_store::{Lease, LeaseCollection};
 
 use self::{
     spec_pool::{ExclusiveSpecPool, KvSpecPool, LeaseSpecPool},
     uncommitted_pool::{ExclusiveUncomPool, KvUncomPool, LeaseUncomPool},
 };
-
+// TODO: Refine code to improve reusability for different conflict pool types
```

Comment:
What does different conflict pool types mean here？ Is there any example?

Reply:
For instance, `KvSpecPool` `LeaseSpecPool` `KvUncomPool` and`LeaseUncomPool` might have some redundant code.